interfaces/openfga/v0/schema.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+"""This file defines the schemas for the provider and requirer sides of the openfga interface.
+
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"openfga":
+                 {"address":  "10.10.0.17",
+                  "port": "8080",
+                  "scheme": "http",
+                  "token": "test-token",
+                  "store_id": "01GK13VYZK62Q1T0X55Q2BHYD6",
+                 }
+             }
+
+    RequirerSchema:
+        unit: <empty>
+        app: {"store_name": "test-store-name"}
+"""
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel
+
+
+class OpenFGAProviderData(BaseModel):
+    address: str # The address of the OpenFGA service.
```

Comment:
you can use `pydantic.Field` instead of raw python type annotations in order to attach a description to a field, and even example values if you want to

Reply:
done